harc_game_web/apps/bank/models.py

```diff
@@ -51,6 +52,10 @@ class Bank(models.Model):
     def task(self):
         return self.documented_task.task
 
+    def score(self):
+        score = Bank.objects.filter(accrual_deleted=False).filter(user=self.user).aggregate(Sum('accrual'))['accrual__sum']
+        return score if score != None else 0
```

Comment:
poprawnie pythonowo powinno być `score is not None` (a nie `!=`)

Reply:
OK, poprawie